src/App.js

```diff
@@ -0,0 +1,26 @@
+import {
+  BrowserRouter, Routes, Route,
+} from 'react-router-dom';
+import './App.css';
+
+import BookForm from './components/BookForm';
+import Categories from './components/Categories';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <>
+
+      <BrowserRouter>
+        <Navbar />
```

Comment:
Great job @francksefu 🤝

- [ ] You have done well to create this Navabar component as requested by the previous reviewer, however, this component is empty so it does not contain its intended functionality. Kindly ensure this <Navbar>,  contains 2 <Link>s. One to / and one to /categories.
- [ ] Try to check properly if your project runs as expected before you submit for the re-review.